### PR TITLE
Lk warp contribution guide

### PR DESCRIPTION
**Pull request by ekiernan**
* Added a WARP Contribution Guide
* Added an Example Contribution tutorial

### CODE REVIEW

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Contribution guidelines for new pipelines are in development, but to contribute a new pipeline, reach out to the WARP team first by [filing an issue](https://github.com/broadinstitute/warp/issues/new). Additionally, read our [Best Practices](https://broadinstitute.github.io/warp/docs/About_WARP/BestPractices) for pipeline development.
+:::
+
+### Contributor agreement
+All user contributions fall under a [BSD 3-Clause License](https://github.com/broadinstitute/warp/blob/master/LICENSE), are not copyright, and are open source. When you make a contribution, you attest that all contributions are your own and not copyrighted by another party.
+
+
+## Contribution Guidelines
+#### 1. Scope the work.
+Contributions can range from tiny syntax changes to large task, workflow, or Docker code updates. Often changes to a single task WDL or Docker file can affect multiple workflows that import those same task WDLs or Dockers. This is why it's important to scope updates before proceeding.
```
Comment (wjdingman1) [inline website/docs/contribution/contribute_to_warp/contribution-guidelines.md:20]:
```suggestion
Contributions can range from tiny syntax changes to large task, workflow, or Docker updates. Often changes to a single task WDL or Dockerfile can affect multiple workflows that import those same task WDLs or Docker images. This is why it's important to scope updates before proceeding.
```

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+If you **do not** have WARP permissions (i.e. you can’t make a branch off of the develop branch), make a fork of the WARP repository following the GitHub Docs [instructions for forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+![](fork.png)
+
+#### 4. Make your updates on your WARP branch or fork. 
+As your work progresses, make commits to your WARP branch or fork. 
+
+#### 5. Assess and make updates to accompanying pipeline files.
+
+Updates to pipeline code, Dockers, or reference files might require coordinating additional updates both in WARP, such as modifying accompanying changelogs and pipeline overviews, or outside of WARP, such as uploading Dockers or reference files to public cloud repositories.
```
Comment (wjdingman1) [inline website/docs/contribution/contribute_to_warp/contribution-guidelines.md:49]:
```suggestion
Updates to pipeline code, Docker images, or reference files might require coordinating additional updates both in WARP, such as modifying accompanying changelogs and pipeline overviews, or outside of WARP, such as uploading Docker images or reference files to public cloud repositories.  
```

**Review by wjdingman1 — APPROVED** [review]
Looks great! Just made some small changes with language around docker stuff

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+* All changes that affect pipeline workflows must be recorded in the pipeline(s) changelog(s) using semantic versioning. An explanation of this can be found in the [changelog style guide](https://broadinstitute.github.io/warp/docs/contribution/contribute_to_warp/changelog_style). 
+
+* When you update the changelog, the pipeline’s WDL workflow version number must also match the changelog entry to ensure the updates pass the WARP testing process (this is shown in the [Example Contribution](#example-contribution-process)).
+
+* If you make any changes to files hosted in cloud repositories, like pipeline Docker images or reference files, coordinate with the WARP team during or before the review process to push the updated files to cloud storage. 
+
+* Flag or make necessary updates to the accompanying pipeline documentation such as the pipeline [overviews in WARP Documentation](https://broadinstitute.github.io/warp/).
```
Comment (gbggrant) [inline website/docs/contribution/contribute_to_warp/contribution-guidelines.md:57]: A little unclear here.  You mean to have them not to forget to add changes to the overview?

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+#### 6. Make a pull request to the WARP **develop** branch. 
+
+When updates are ready, [make a pull request (PR)](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) using WARP's **develop** branch as a base. In the comments section of the PR, notify the WARP team to review the PR and start the necessary repository [tests](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines). You can do this by typing `test and review @broadinstitute/warp-admins`.
+
+#### 7. Revise the PR as necessary for the [review process](#review-process). 
+
+#### 8. Merge updates to the WARP **develop** branch. 
+
+If you have permission to merge to the WARP repository, you can merge the PR once it's approved. You’ll receive a reminder to perform a [“squash merge”](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits). Please delete individual commit names and make one summary name for all commits.
```
Comment (gbggrant) [inline website/docs/contribution/contribute_to_warp/contribution-guidelines.md:69]:
```suggestion
If you have permission to merge to the WARP repository, you can merge the PR once it's approved. You’ll receive a reminder to perform a [“squash merge”](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits). Please delete individual commit comments and make one summary comment for all commits. 
```

File: website/docs/contribution/contribute_to_warp/contribution-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+![](PR_base.png) 
+
+
+After the PR is created, we can tag the WARP team by making a comment in the PR: `test and review @broadinstitute/warp-admins`.
+
+![](warp-admins-tag.png)
+
+The team will then work with us to kick off any necessary [tests](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines).
+When the tests are successful, we'll see a green check left of the test name on the PR's Conversation tab in the GitHub UI.
```
Comment (gbggrant) [inline website/docs/contribution/contribute_to_warp/contribution-tutorial.md:67]:
```suggestion
When the tests are successful, we'll see a green check to the left of the test name on the PR's Conversation tab in the GitHub UI.
```

File: website/docs/contribution/contribute_to_warp/contribution-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+![](tests.png)
+
+Even if the tests pass, merging will be blocked until the review is complete and at least two reviewers have approved the PR, as outline in the [Contribution Guidelines](contribution-guidelines.md/#review-process).
```
Comment (gbggrant) [inline website/docs/contribution/contribute_to_warp/contribution-tutorial.md:71]:
```suggestion
Even if the tests pass, merging will be blocked until the review is complete and at least two reviewers have approved the PR, as outlined in the [Contribution Guidelines](contribution-guidelines.md/#review-process).
```

**Review by gbggrant — APPROVED** [review]
Looks good!

**Review by jessicaway — APPROVED** [review]
A couple minor comments, but overall looks good! Thank you Liz!

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md (hunk 3 above)
Comment (ekiernan) [reply website/docs/contribution/contribute_to_warp/contribution-guidelines.md]: I updated it to be more explicit to change the documentation if necessary.

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+If you have permission to merge to the WARP repository, you can merge the PR once it's approved. You’ll receive a reminder to perform a [“squash merge”](https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits). Please delete individual commit names and make one summary name for all commits. 
+
+If you don’t have WARP permissions, the WARP team will merge the PR for you when it’s ready.
+
+## Review Process
+
+### WARP review process and requirements
+After contributing a PR, a WARP team member will start a series of [tests](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines), usually by making an “Ok to test” comment.
+
+For each PR, WARP requires approval from a minimum of two reviewers, often an engineer, a scientific owner, and sometimes a clinical owner. All comments and requests for changes are made directly in the GitHub PR.
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/contribution/contribute_to_warp/contribution-guidelines.md]: Updated this.

File: website/docs/contribution/contribute_to_warp/contribution-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+## Review Process
+
+### WARP review process and requirements
+After contributing a PR, a WARP team member will start a series of [tests](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines), usually by making an “Ok to test” comment.
+
+For each PR, WARP requires approval from a minimum of two reviewers, often an engineer, a scientific owner, and sometimes a clinical owner. All comments and requests for changes are made directly in the GitHub PR. 
+
+Comments will be likely be made within **3 business days** of the PR submission. If no review is started by that time, tag @Kylee Degatano in the PR. 
+
+If a PR is abandoned after starting the review process, the WARP team will either take ownership of it or close the PR.
```
[the remark this answers is not in the document]
Comment (ekiernan) [reply website/docs/contribution/contribute_to_warp/contribution-guidelines.md:82]: Good question! I'll ask the team about this and we can update.

**Pull request merged**